task-5/task-5/src/main.ts

```diff
@@ -0,0 +1,5 @@
+import './style.css'
+import './normalize.css'
```

Comment:
**Normalize needs to come before your styles always**, this way it's not doing it's job

Reply:
Mistake as last time :)